README.md

```diff
@@ -1,105 +1,88 @@
-# Webpack Starter Kit
-
-## Clone This Repo
-
-That's right, _clone_ not fork. You will use this repo multiple times, but you can only fork a repository once. So here is what you need to do to clone the repo and still be able to push changes to your repo:
-
-1. Clone down this repo. Since you don't want to name your project "webpack-starter-kit", you can use an optional argument when you run `git clone` (you replace the `[...]` with the terminal command arguments): `git clone [remote-address] [what you want to name the repo]`
-1. Remove the default remote: `git remote rm origin` (notice that `git remote -v` not gives you back nothing)
-1. Create a new repo on GitHub with the name of `[what you want to name the repo]` to be consistent with naming
-1. Copy the address that you would use to clone down this repo - something like `git@github.com:...`
-1. Add this remote to your cloned down repo: `git remote add origin [address you copied in the previous step]` - do not include the brackets
-
-Now try to commit something and push it up to your new repo. If everything is setup correctly, you should see the changes on GitHub.
-
+# Refactor Tractor: What's Cookin'?
+
+Do you like eating?  If so, try cooking. And we can help!
+
+Our site, 'What's Cookin'?', displays a variety of recipes to tempt your taste-buds. A user is free to:
+ - puruse the recipes
+ - search recipes by keyword
+ - search recipes by types/tags
+ - add interesting recipes to a list of recipes 'to cook'
+ - add tasty recipes to a list of recipe 'favorites'
+ - keep track of ingredients in a 'pantry'
+ - search for ingredients in the 'pantry'
+ - add ingredients to the 'pantry'
+ - see what recipes can be made using exisiting 'pantry' ingredients
+ - for each recipe, see which ingredients are missing from the 'pantry'
+ - 'cook' a recipe, and see the pantry amounts reflect the ingredients used
+ - display the cost of the recipes with very real and accurate prices
+
+This is a group project for [Turing School](https://turing.io/) in Module 2, which introduced us to working with pre-existing code in preparation for any legacy codebases we may work with in the future. 
+
+We recieved some existing code and were able to refactor and build out the functionality, with emphasis on DRY logic and accessibility, as well as using dynamic fetch requests to API endpoints to reflect a strong data model.
```

Comment:
it feels like this will fit well in the overview, maybe moving up a little higher as the intro to the project?

Reply:
Moved!